scripts/build/gen_isr_tables.py

```diff
@@ -13,7 +13,7 @@
 from elftools.elf.elffile import ELFFile
 from elftools.elf.sections import SymbolTableSection
 
-ISR_FLAG_DIRECT = (1 << 0)
+ISR_FLAG_DIRECT = 1 << 0
```

Comment:
why?

Reply:
This is needed to pass compliance checks. Otherwise it fails with:

```
C0325:Unnecessary parens after '=' keyword (superfluous-parens)
File:scripts/build/gen_isr_tables.py
Line:16
Column:0
C0325:Unnecessary parens after '=' keyword (superfluous-parens)
File:scripts/build/gen_isr_tables.py
Line:358
Column:0
Error: Process completed with exit code 1.
```

Reply:
odd. In that case this fix should be in a different PR.

Reply:
Ok, dropping that for now.